ui/app/AppLayouts/Wallet/LeftTab.qml

```diff
@@ -5,8 +5,10 @@ import QtGraphicalEffects 1.13
 import "../../../imports"
 import "../../../shared"
 import "./components"
+import "../../../shared/polyfill.number.toLocaleString.js" as NumberPolyFill
```

Comment:
It might be easier to just put the `toLocaleString` in `Utils`, that way we don't have to import this in all the files that need it